app/services/weather_service.rb

```diff
@@ -0,0 +1,14 @@
+class WeatherService
+  def self.get_weather(lat, lon)
+    response = conn.get('/api/v1/forecast') do |f|
+      f.params[:lat] = lat
+      f.params[:lon] = lon
+    end
+    json = JSON.parse(response.body, symbolize_names: :true)
+  end 
+
+  private
+  def self.conn
+    Faraday.new(url: 'https://weather-microservice-api.herokuapp.com/')
```

Comment:
Ian said that this URL should be an environment variable I believe (also prevents publicly displaying our API that doesn't require auth). We did this in the FE with something like `BE_URL: 'https://solar-garden-be.herokuapp.com`